packages/flutter/test/material/slider_test.dart

```diff
@@ -1949,6 +1949,94 @@ void main() {
     await gesture.up();
   });
 
+  testWidgets('Slider removes value indicator from overlay if Slider gets disposed without value indicator animation completing.', (WidgetTester tester) async {
+    final Key sliderKey = UniqueKey();
+    double value = 0.0;
+
+
+    Widget buildApp({
+      Color activeColor,
+      Color inactiveColor,
+      int divisions,
```

Comment:
Looks like divisions isn't used, and is hardcoded to 10 below, was that intentional?

Reply:
It wasn't intentional a lot of the method use buildApp in the slider and range slider test. To keep it consistent I also used the similar format. I used the divisions variable but it doesn't change anything